### PR TITLE
Bug 1768496: Clarify message about Quay image scanning

### PR DESCRIPTION
**Pull request by spadgett**
cc @shawn-hurley @jwforres @alecmerdler @itsptk

### CODE REVIEW

**Comment by spadgett** [comment]
Looks like the plugin is broken. I'm investigating.

```
TypeError: "props.k8sResult is undefined"
    SecurityBreakdownPopup summary.tsx:42
```

**Comment by spadgett** [comment]
#3374 broke the plugin. release-4.3 is fine since that PR only landed in master.

**Comment by spadgett** [comment]
Dashboards · OKD 2019-11-22 16-29-24

**Comment by spadgett** [comment]
Screen Shot 2019-12-03 at 10 40 09 AM

File: frontend/packages/container-security/src/components/summary.tsx [hunk 1]
```diff
@@ -46,7 +46,8 @@ export const SecurityBreakdownPopup: React.FC<SecurityBreakdownPopupProps> = (pr
   return (
     <>
       <div className="co-overview-status__control-plane-description">
-        Quay analyzes container images to identify vulnerabilities.
+        Container images from Quay are analyzed to identify vulnerabilities. Images from other
+        registries will not be scanned.
```
Comment (rhamilto) [inline frontend/packages/container-security/src/components/summary.tsx]: Grammar nit to make verb parallel with first sentence.

```suggestion
        registries are not scanned.
```

**Comment by spadgett** [comment]
@rhamilto thanks, updated

**Pull request merged**